wac_rules/wac_gameoflife.py

```diff
@@ -0,0 +1,53 @@
+from random import randint
+
+#return if cube is alive in function of its neighbours
+def behaviour_cell(alive, neighbours) -> bool:
```

Comment:
je conseille de commencer les méthodes qui renvoient une valeur par get ou is
is_alive()
get_neighbours_count()
etc. 
On comprend tout de suite comme ça !